src/OrleansAzureUtils/Storage/AzureQueueDataManager.cs

```diff
@@ -109,8 +109,12 @@ public AzureQueueDataManager(string queueName, string deploymentId, string stora
                 var myQueue = queueOperationsClient.GetQueueReference(QueueName);
 
                 // Create the queue if it doesn't already exist.
-
+#if !NETSTANDARD_TODO
+                //BeginCreateIfNotExists and EndCreateIfNotExists Not supported yet. 
                 bool didCreate = await Task<bool>.Factory.FromAsync(myQueue.BeginCreateIfNotExists, myQueue.EndCreateIfNotExists, null);
+#else
+                bool didCreate = true;
```

Comment:
As we talked in private, it makes more sense to not introduce those #if NETSTANDARD_TODO when the fix is trivial